aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -515,27 +579,30 @@ module aptos_framework::account {
         rotate_authentication_key_ed25519(&alice, test_signature, test_signature, pk, pk);
     }
 
+    #[test_only]
+    public entry fun create_account_from_public_key(pk_bytes: vector<u8>): signer {
```

Comment:
A better name: `create_account_from_ed25519_public_key` lest other people think it's okay to call this with different types of PKs.